dashboard/app/views/shared/_school_info_registration.html.haml

```diff
@@ -0,0 +1,101 @@
+- require 'country_codes'
```

Comment:
Might it make sense to re-implement this in React instead of haml, and have one element that takes various props related to layout, required fields, etc? Several of us (@davidsbailey @mehalshah) have been discussing this as a longer term tech debt item since there's a good bit of complexity in jquery state management here. Now that we're extending this and adding more haml, it might be a good time to port it to React.